<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load">
         <property name="text">
          <string>load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="threshold">
         <property name="text">
          <string>threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="thresh_value"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView_2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
